pipelines/snap-atac/snap-atac-primary.wdl

```diff
@@ -0,0 +1,238 @@
+version 1.0
+
+workflow scATAC {
+    input {
+        File input_fastq1
+        File input_fastq2
+
+        String genome_name
+        File input_reference
+
+        String output_bam = "aligned.bam"
+    }
+    parameter_meta {
+        input_fastq1: "read 1 input fastq"
+        input_fastq2: "read 2 input fastq"
+        input_reference: "tar file with BWA reference"
+        output_bam: "output BAM file"
+        genome_name: "name of the genome for snap atac"
+    }
+    call AlignPairedEnd {
+        input:
+            input_fastq1 = input_fastq1,
+            input_fastq2 = input_fastq2,
+            input_reference = input_reference,
+            output_bam = output_bam
+    }
+    call SnapPre {
+        input:
+            input_bam = AlignPairedEnd.aligned_bam,
+            output_snap_basename = 'output.snap',
+            genome_name = genome_name,
+            input_reference = input_reference,
+    }
+    call SnapCellByBin {
+        input:
+            snap_input = SnapPre.output_snap,
+            bin_size_list = "10000"
+    }
+    call MakeCompliantBAM {
+        input:
+            input_bam = AlignPairedEnd.aligned_bam
+    }
+    call BreakoutSnap {
+        input:
+            snap_input = SnapCellByBin.output_snap
+    }
+    output {
+        File output_snap_qc = SnapPre.output_snap_qc
+        File output_snap = SnapCellByBin.output_snap
+        File output_aligned_bam = MakeCompliantBAM.output_bam
+        File breakout_barcodes = BreakoutSnap.barcodes
+        File breakout_fragments = BreakoutSnap.fragments
+        File breakout_binCoordinates = BreakoutSnap.binCoordinates
+        File breakout_binCounts = BreakoutSnap.binCounts
+        File breakout_barcodesSection = BreakoutSnap.barcodesSection
+    }
+}
+
+task AlignPairedEnd {
+    input {
+        File input_fastq1
+        File input_fastq2
+        File input_reference
+        String reference_unpack_name = "genome/genome.fa"
+        String output_bam
+        Int min_cov = 0
+        String docker_image = "quay.io/humancellatlas/snaptools:0.0.1"
+    }
+
+    Int num_threads = 16
+    Float input_size = size(input_fastq1, "GiB") + size(input_fastq2, "GiB") + size(input_reference, "GiB")
+
+    command {
+        set -euo pipefail
```

Comment:
Question: I've started to see `-u` cropping up in bash scripts used in cromwell. What's the purpose behind it?